sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/TransactionalBatchPatchItemRequestOptions.java

```diff
@@ -0,0 +1,59 @@
+package com.azure.cosmos;
+
+import com.azure.cosmos.implementation.RequestOptions;
+import com.azure.cosmos.util.Beta;
+
+@Beta(value = Beta.SinceVersion.V4_11_0, warningText = Beta.PREVIEW_SUBJECT_TO_CHANGE_WARNING)
+public class TransactionalBatchPatchItemRequestOptions extends TransactionalBatchItemRequestOptions {
+    private String filterPredicate;
+
+    /**
+     * Gets the FilterPredicate associated with the request in the Azure Cosmos DB service.
+     *
+     * @return the FilterPredicate associated with the request.
+     */
+    public String getFilterPredicate() {
+        return this.filterPredicate;
+    }
+
+    /**
+     * Sets the FilterPredicate associated with the request in the Azure Cosmos DB service.
+     *
+     * @param filterPredicate the filterPredicate associated with the request.
+     * @return the current request options
+     */
+    public TransactionalBatchPatchItemRequestOptions setFilterPredicate(String filterPredicate) {
+        this.filterPredicate = filterPredicate;
+        return this;
+    }
+
+    /**
+     * Sets the If-Match (ETag) associated with the operation in TransactionalBatch.
+     *
+     * @param ifMatchETag the ifMatchETag associated with the request.
+     * @return the current request options
+     */
+    @Beta(value = Beta.SinceVersion.V4_7_0, warningText = Beta.PREVIEW_SUBJECT_TO_CHANGE_WARNING)
```

Comment:
you have mixed versions for the beta since. all should be on the same new version.
some places you have `V4_11_0` and some places you have `V4_11_0`
please fix here and everywhere else.